File: 백준/Bronze/2738. 행렬 덧셈/행렬 덧셈.py
n, m = map(int, input().split())
arr, arr1 = [], []

for i in range(n):
        temp = list(map(int, input().split()))
        arr.append(temp)

for i in range(n):
        temp = list(map(int, input().split()))
        arr1.append(temp)


for i in range(n):
    for j in range(m):
        print(arr[i][j] + arr1[i][j], end = ' ')
    print()

File: 백준/Bronze/2738. 행렬 덧셈/행렬 덧셈.cc
#include <iostream>
#include <algorithm>
using namespace std;

int main()
{
    int n, m;
    cin >> n >> m;
    int arr[n][m], arr1[n][m];
    for(int i = 0; i < n; i++){
        for(int j =0; j< m; j++){
            cin >> arr[i][j];
        }
    }

    for(int i = 0; i < n; i++){
        for(int j =0; j< m; j++){
            cin >> arr1[i][j];
        }
    }

    for(int i = 0; i < n; i++){
        for(int j =0; j< m; j++){
            cout << arr[i][j] + arr1[i][j] << ' ';
        }
        cout << endl;
    }
    
}